src/Login/Login.js

```diff
@@ -22,15 +23,19 @@ class Login extends Component {
     this.setState({
       email: '',
       password: '',
-      error: ''
+      // error: ''
     })
   }
 
   handleLogin = (event) => {
     event.preventDefault();
-    getUserData(this.state.email, this.state.password)
-    .then(user => this.props.setUser(user))
-    .catch(error => this.setState({ error: error }))
+    if (this.state.email !== '' && this.state.password !== '') {
```

Comment:
Nice error handling here.  I appreciate that this condition is added here, since we don't want to make the fetch request if we're certain the user has missing information.